torchtune/models/gemma/_component_builders.py

```diff
@@ -307,14 +307,14 @@ def lora_gemma_self_attention(
     )
     output_proj = (
         LoRALinear(
-            embed_dim,
+            num_heads * head_dim,
             embed_dim,
             rank=lora_rank,
             alpha=lora_alpha,
             quantize_base=quantize_base,
         )
         if "output_proj" in lora_modules
-        else nn.Linear(embed_dim, embed_dim, bias=False)
+        else nn.Linear(num_heads * head_dim, embed_dim, bias=False)
```

Comment:
@ebsmothers here there was another "silent" error since `embed_dim = num_heads * head_dim` for gemma2b but this is not true for gemma7b.